### PR TITLE
feat(mojaloop/2867): switch as fspiop source

### PR DESCRIPTION
**Pull request by vijayg10**
feat([mojaloop/2867](https://github.com/mojaloop/project/issues/2867)): POST /bulkTransfers from Switch --> PayeeFSP to use Switch as fspiop-source

### CODE REVIEW

File: docker/wait-for/wait-for-objstore.sh [hunk 1]
```diff
@@ -4,9 +4,4 @@ echo "** STARTUP - Checking for Object Store connection..."
 
 source /opt/wait-for/wait-for.env
 
-apk add --no-cache mongodb
```
Comment (mdebarros) [inline docker/wait-for/wait-for-objstore.sh:7]: How does this work if the mongo client isnt installed?

File: src/handlers/notification/index.js [hunk 2]
```diff
@@ -187,6 +188,8 @@ const processMessage = async (msg, span) => {
     const payloadForCallback = decodedPayload.body.toString()
     let callbackHeaders
 
+    let jwsSigner = getJWSSigner(from)
```
Comment (mdebarros) [inline src/handlers/notification/index.js:191]: This seems unecessary...as we would only create a JWSSigner when the message is sent from the Switch (which is done below).

i.e. why wouldn't all the messages here be signed by the switch? :D

For example why do we not sign the message on https://github.com/mojaloop/bulk-api-adapter/pull/90/files#diff-2ad92add22f92879b719bfa1dba689027c5282e313802338863d88571f0293c1R227 or any of the other alternative scenarios here?

**Review by mdebarros — CHANGES_REQUESTED** [review]
Looking pretty much there, just a few comments for you to address.

File: src/handlers/notification/index.js (hunk 2 above)
Comment (vijayg10) [reply src/handlers/notification/index.js]: I added the jws signing for all the messages as we discussed. @elnyry-sam-k please review this.

File: docker/wait-for/wait-for-objstore.sh (hunk 1 above)
Comment (vijayg10) [reply docker/wait-for/wait-for-objstore.sh:7]: Hi @mdebarros , the wait-for script we are using checks only if the TCP port is opened or not. We are doing the same for kafka. So we don't need the mongodb client here.

**Review by mdebarros — APPROVED** [review]
+1

**Pull request merged**